### PR TITLE
zh-cn prepare to convert h2m abortsignal

### PR DESCRIPTION
**Pull request by fwqaaq**

### CODE REVIEW

File: files/zh-cn/web/api/abortsignal/index.md [hunk 1]
```diff
@@ -2,32 +2,33 @@
...
-  - DOM
-  - Experimental
-  - Interface
-  - Reference
+- API
+- AbortSignal
+- DOM
+- Experimental
+- Interface
+- Reference
```
Comment (yin1999) [inline files/zh-cn/web/api/abortsignal/index.md:10]:
```suggestion
  - API
  - AbortSignal
  - DOM
  - Experimental
  - Interface
  - Reference
```

**Review by yin1999 — CHANGES_REQUESTED** [review]
You haven't convert those documents to markdown

**Comment by Jack-Zhang-1314** [comment]
I don't know why this is so

**Comment by yin1999** [comment]
> I don't know why this is so 

Please run command in `powershell` or `cmd`. Are you using unix like environment in windows?

**Comment by Jack-Zhang-1314** [comment]
Maybe for windows computers it should be restart

!

**Comment by yin1999** [comment]
I'm not sure about the problem, it works fine for me on windows.

**Comment by Jack-Zhang-1314** [comment]
> I'm not sure about the problem, it works fine for me on windows.

When I restarted, I failed again. Now you can only turn it off.

**Comment by Jack-Zhang-1314** [comment]
It is not a competent computer

**Comment by yin1999** [comment]
> When I restarted, I failed again. Now you can only turn it off.

Closing this

**Pull request closed without merging**